src/libtrx/include/libtrx/game/output/types.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <stdint.h>
+
+typedef struct {
+    uint16_t u;
+    uint16_t v;
+} PHD_UV;
```

Comment:
Nitpick, can we rename this to `TEXTURE_UV` while we're at it?

Reply:
Done